### PR TITLE
[Qwen3] Add 1.7b Config

### PR DESCRIPTION
**Pull request by wwwjn**
Contents:
1. Minor fix in qwen 0.6b training config
2. Add qwen 1.7b training cofnig
3. Fix the number of parameters calculation when weight tying is enabled

### CODE REVIEW

**Review by Jack-Khuu — APPROVED** [review]
Thanks for the quick add!!

File: torchtitan/experiments/qwen3/train_configs/qwen3_1.7b.toml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+[model]
+name = "qwen3"
+flavor = "1.7B"
+hf_assets_path = "./assets/hf/Qwen3-1.7B"
+# converters = ["float8"]
+
+[optimizer]
+name = "AdamW"
+lr = 3e-4
```
Comment (tianyu-l) [inline torchtitan/experiments/qwen3/train_configs/qwen3_1.7b.toml:23]: We need to verify this and learning rate with some real training job say for 3k steps, unless there is some "official" numbers we can refer to.

I don't mind landing it for now to unblock and do the studies later.

File: torchtitan/experiments/qwen3/model/args.py [hunk 2]
```diff
@@ -64,4 +64,9 @@ def get_nparams_and_flops(self, model: nn.Module, seq_len: int) -> tuple[int, in
         )
         num_flops_per_token = 6 * (nparams - nparams_embedding) + 12 * l * h * q * t
 
+        if self.enable_weight_tying:
+            nparams = nparams - sum(
+                p.numel() for p in model.tok_embeddings.parameters()
```
Comment (tianyu-l) [inline torchtitan/experiments/qwen3/model/args.py]: could you reuse `nparams_embedding`?

**Review by tianyu-l — APPROVED** [review]
please reuse `nparams_embedding` for nparams computation.

File: torchtitan/experiments/qwen3/train_configs/qwen3_1.7b.toml (hunk 1 above)
Comment (wwwjn) [reply torchtitan/experiments/qwen3/train_configs/qwen3_1.7b.toml:23]: Sure, I will leave a TODO in README to capture this
